src/Components/Filters.test.js

```diff
@@ -5,7 +5,40 @@ import Filters from './Filters';
 
 describe('<Filters />', () => {
   it('renders correctly', () => {
-    const tree = renderer.create(<Filters initialData={[]} onConfirm={jest.fn()} onFiltered={jest.fn()} />).toJSON();
+    const drills = [
```

Comment:
I would move testdata into a separate file called something along the lines of  `testFactory`. Then we can reuse the same datastructures for other tests a lot more easily.

When you setup the factory make sure to make all values configurable like so:
```
export const createDrill = ({ id = 10, type = "fitness", [...]}) => ({
    id,
    type,
    [...]
});
```

Reply:
Totally agree!

Reply:
I did it on moby's pull request #45 
I'll merge it here when it's merged